testdata/ingress_rg_shared_alb/input/k8s/01-ing.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: myingress
```

Comment:
wha does the routegroup has `    zalando.org/aws-load-balancer-shared: "true"` but ingress not?

Reply:
I don't know the original idea but in theory it should not make any difference since if not specified the ingress-controller will create only one LB for both RG and ingress.

I don't have strong opinions about this, I would not mind removing it or letting it there to make it more explicit.

Reply:
I'll make both have the same annotations.